### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#604] [Galaxy Trivia] [answered]
⬜🟦✡️
🟦 x6 ⭐
🟥🟦⬜🦅

**Comment by peterbaccaro** [comment, marked as answer]
1) ISRAEL
2) KOSOVO
3) SERBIA

![flags3](https://user-images.githubusercontent.com/74666079/233705686-4b06c713-9823-4fb9-88c1-b551a3038846.png)
